.github/workflows/Integration.yml

```diff
@@ -52,6 +52,11 @@ jobs:
           - Lux
           - SciML
           - KernelAbstractions
+          - Molly
+        exclude:
+          - version: '1.10'
+            os: linux-x86-n2-32
```

Comment:
It's unnecessary to specify also the `os` (and that's slightly fragile in case we change runners)
```suggestion
```

Reply:
Wait, no, my suggestion was right!  I misread the exclusion, I thought we wanted to exclude 1.11, and I saw the 1.11 job running, but we're actually excluding v1.10!